readme.txt

```diff
@@ -205,6 +205,10 @@ Check out our extensive [knowledgebase](https://evnt.is/18wm) for articles on us
 
 == Changelog ==
 
+= [5.13.0.3] 2024-08-01 =
```

Comment:
Should this be in a separate changelog file?

Reply:
Im just skipping a few steps - cause it will probably go directly to release after approved and pass QA